README.md

```diff
@@ -23,22 +23,22 @@ There are multiple ways to install bigquery-grafana go to [INSTALL](https://raw.
  * Table view
  * Annotations
  * BQ queries in variables
- * Sharded tables (`tablename_YYYYMMDD`)
+ * Shared tables (`tablename_YYYYMMDD`)
```

Comment:
Is `shared tables` the correct term? aren't those sharded (as in: shard) tables?

Reply:
Of course it's Sharded. Thanks for spotting it!